news/search.py

```diff
@@ -57,13 +57,13 @@ def search(request):
     if ('q' in request.GET) and request.GET['q'].strip():
         query_string = request.GET['q']
 
-        vector = SearchVector('title', weight='A') + SearchVector('subheading', weight='B') + SearchVector('author', weight='B') + SearchVector('body', weight='C') + SearchVector('tags__name', weight='C')
+        vector = SearchVector('title', weight='A') + SearchVector('article_subjects__name', weight='B') + SearchVector('body', weight='B') + SearchVector('author', weight='B') + SearchVector('tags__name', weight='C')  + SearchVector('collections__name', weight='C')  + SearchVector('content_types__content_type', weight='C')
```

Comment:
is it intended to have `weight=C` on the last three vectors?

Reply:
Yes, I am giving them the same weight. There are specific searches for those fields on the blog page. I'm assuming someone using the search box is looking for something different, but added them in case there is a match. This is all a guessing game as to what will help improve the search results.